src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs

```diff
@@ -140,6 +140,19 @@ public class CommitmentOrchestrator : BaseCommitmentOrchestrator
             };
         }
 
+        public async Task<TransferFundedViewModel> GetAllTransferFunded(long providerId)
+        {
+            var data = (await GetAllCommitmentsWithTheseStatuses(providerId, new [] { RequestStatus.WithSenderForApproval, RequestStatus.RejectedBySender })).ToList();
+            _logger.Info($"Provider getting all transfer funded ({data.Count}) :{providerId}", providerId);
```

Comment:
The log message could be improved to use the correct tense, as they have already been fetched, e.g. "Got {data.Count} transfer funded cohorts for provider {providerId}"

Reply:
I'll leave in the existing format. AS it's in this format for teh other similar methods